src/applications/edu-benefits/tests/5490/schema/expected-data/previous-sponsor-checked.json

```diff
@@ -15,13 +15,6 @@
     },
     "vaFileNumber": "C12345678"
   },
-  "veteranFullName": {
```

Comment:
expected results in unit tests now only expect `veteranFullName` under `previousBenefits`